tests/test_staging/test_validation.py

```diff
@@ -11,11 +12,15 @@
     validate_data_with_schema,
     combine_schemas_validate_full_df,
 )
-from src.utils.helpers import Config_settings
+
 
 # Get the config
-conf_obj = Config_settings()
-config = conf_obj.config_dict
+def generate_config(val):
```

Comment:
I've made some minor changes to the tests that used the previous config. It's all fixed now.